src/Engine/Manager/ScriptManager.cpp

```diff
@@ -660,9 +661,11 @@ void ScriptManager::Update(World& world, float deltaTime) {
 
                 if (script->IsInPropertyMap(name, typeId)) {
 
-                    if (typeId == engine->GetTypeIdByDecl("Entity@"))
+                    if (typeId == engine->GetTypeIdByDecl("Entity@")) {
+                        Entity* whatever = Hymn().GetEntityByGUID(*(unsigned int*)script->GetDataFromPropertyMap(name));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'whatever' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)